docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,152 @@ sidebar_position: 2
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+## Prerequisites
+
+Ensure you have installed and configured these:
+
+- Kubernetes cluster (Use [minikube](https://minikube.sigs.k8s.io/) or
+  [kind](https://kind.sigs.k8s.io/) locally)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Service Binding Operator](installing-service-binding).
+
+## Database Backend
+
+The application is going to connect to a PostgreSQL database backend.  The
+PostgreSQL can be setup using the [Crunchy PostgreSQL operator from
+OperatorHub.io][crunchy].
+
+The installation of the operator doesn't create a database instance for
+connection.  To create a database instance, you need to create custom resource
+(`Pgcluster`) and that will trigger the operator reconciliation.  For
+convenience, run this command to create `Pgcluster` custom resource:
+
+```bash
+kubectl apply -f https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/pgcluster.yaml
+```
+
+Ensure all the pods in `petclinic-demo` is running (it will take few minutes):
+
+```bash
+kubectl get pod -n petclinic-demo
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+The database will be empty by default, it requires the schema and sample data to
+work with the application.  Now you can initialize the database with the schema
+and sample data using this command:
+
+```bash
+bash <(curl -s https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/init-database.sh)>
+```
+
+Now the database is ready to connect from application.  The next section
+explains how to configure application:
+
+## Application Deployment
+
+Now you can deploy the `spring-petclinic-rest` app with this `Deployment`
+configuration:
+
+```bash
+kubectl apply -f https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/app-deployment.yaml
+```
+
+You can port-forward the application port and try to access it from your local system
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n petclinic-demo
+```
+
+You can open [http://localhost:9966/petclinic](http://localhost:9966/petclinic)
+
+You should see a [Swagger UI][swagger] where you can play with the API.
+
+Since the binding is not present in the application, you will see be any values in results.
+In the next section, you will see how to fix it.
+
+## Service Binding
+
+The application was not working as the bindings were not present in the app.
+
+Now you can create the ServiceBinding custom resource to inject the bindings:
```

Comment:
Suggestion: Please put these instructions as steps. Steps for connecting the application to a backing service.

Put the heading for this section as "Connecting the application to a backing service" and link this section to the 4th bullet item mentioned in the 1st paragraph under "Quick start". I will review again after the rewrite.

Reply:
Done. 34a6d23f94033d82bb896681fab523ea9dc5b9a0